pkg/cfn/manager/delete_tasks.go

```diff
@@ -106,7 +106,7 @@ func (c *StackCollection) NewTasksToDeleteNodeGroups(shouldDelete func(string) b
 func (c *StackCollection) NewTasksToDeleteOIDCProviderWithIAMServiceAccounts(oidc *iamoidc.OpenIDConnectManager, clientSetGetter kubernetes.ClientSetGetter, wait bool) (*TaskTree, error) {
 	tasks := &TaskTree{Parallel: false}
 
-	saTasks, err := c.NewTasksToDeleteIAMServiceAccounts(deleteAll, oidc, clientSetGetter, true)
+	saTasks, err := c.NewTasksToDeleteIAMServiceAccounts(deleteAll, oidc, clientSetGetter, wait)
```

Comment:
Actually, that was intentional. We have no option but to wait, otherwise we cannot proceed. The argument only applies to the deletion of the control plane.

Reply:
Ah, I thought the intent was to pass it. We should probably add a comment here.